src/routes/+page.svelte

```diff
@@ -1,2 +1,22 @@
-<h1>Welcome to SvelteKit</h1>
-<p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
+<script>
+
+</script>
+
+<form action="#">
```

Comment:
Why do we need a form?
When should we use form and when a simple button will do? Can you please explore this?

Reply:
I will explore this, i thinked that we have a multiple files and we want to send them to server, so form is created for this reason ,isnt it?

Reply:
Yes, i read about it and agree with that the form not necessary

Reply:
We send only request to a web server. After that request SPA will upload files directly to S3.